x/evm/keeper/config.go

```diff
@@ -65,12 +66,17 @@ func (k *Keeper) EVMBlockConfig(ctx sdk.Context, chainID *big.Int) (*EVMBlockCon
 	if v != nil {
 		return v.(*EVMBlockConfig), nil
 	}
-
+	time := ctx.BlockHeader().Time
+	blockTime := uint64(time.Unix())
+	blockNumber := big.NewInt(ctx.BlockHeight())
```

Comment:
seems block time might differ between [abci](https://github.com/crypto-org-chain/ethermint/blob/d874df7ebe73a4811f2158756e8098136a7ac083/x/evm/keeper/abci.go#L27) and [grpc](https://github.com/crypto-org-chain/ethermint/blob/d874df7ebe73a4811f2158756e8098136a7ac083/x/evm/keeper/grpc_query.go#L249) call